doc/users/next_whats_new/subplots_adjust.rst

```diff
@@ -0,0 +1,12 @@
+subplots_adjust has a new ``kwarg``: ``rc_default``
+---------------------------------------------------
+
+`.Figure.subplots_adjust` and `.pyplot.subplots_adjust` have a new ``kwarg``:
+``rc_default`` that determines the default values for the subplot parameters.
+
+The `.figure.SubplotParams` object has a new get method
```

Comment:
```suggestion
The `.gridspec.SubplotParams` object has a new get method
```

This has moved canonical location (and is part of why the doc build is failing)

The second reference on line 10 is also flagging, and may need scoping.